Packs/CrowdStrikeFalconSandbox/Integrations/CrowdstrikeFalconSandboxV2/CrowdstrikeFalconSandboxV2.py

```diff
@@ -0,0 +1,532 @@
+
+
+@poll('cs-falcon-sandbox-scan')
+def crowdstrike_scan_command(client: Client, args: Dict[str, Any]) -> PollResult:
+    hashes = args['file'].split(',')
+    scan_response = client.scan(hashes)
+
+    def file_with_bwc_fields(res):
+        return CommandResults(readable_output=' ', indicator=BWCFile(res, {
+            'sha1': 'SHA1',
+            'sha256': 'SHA256',
+            'md5': 'MD5',
+            'job_id': 'JobID',
+            'environment_id': 'environmentId',
+            'threat_score': 'threatscore',
+            'environment_description': 'environmentDescription',
+            'submit_name': 'submitname',
+            'url_analysis': 'isurlanalysis',
+            'interesting:': 'isinteresting',
+            'vx_family': 'family'}, False, size=res['size'], file_type=res['type'], sha1=res['sha1'],
+            sha256=res['sha256'], md5=res['md5'], sha512=res['sha512'], name=res['submit_name'],
+            ssdeep=res['ssdeep'], malware_family=res['vx_family'],
+            dbot_score=get_dbot_score(res['sha256'], res['threat_level'])))
+
+    command_result = [CommandResults(outputs_prefix='CrowdStrike.Report',
+                                     raw_response=scan_response, outputs=scan_response,
+                                     readable_output=create_scan_results_readable_output(scan_response)),
+                      *[file_with_bwc_fields(res) for res in scan_response]]
+    if len(scan_response) != 0:
+        return PollResult(command_result)
+    try:
+        if len(hashes) == 1:
+            key = get_api_id(args)
+            return PollResult(continue_to_poll=lambda: not has_error_state(client, key), response=command_result)
+    except ValueError:
+        demisto.debug(f'Cannot get a key to check state for {hashes}')
+    return PollResult(continue_to_poll=True, response=command_result)
+
+
+def crowdstrike_analysis_overview_summary_command(client: Client, args: Dict[str, Any]):
+    result = client.analysis_overview_summary(args['file'])
+    return CommandResults(
+        outputs_prefix='CrowdStrike.AnalysisOverviewSummary',
+        outputs_key_field='sha256',
+        outputs=result,
+        raw_response=result,
+        readable_output=tableToMarkdown('Analysis Overview Summary:', result,
+                                        headerTransform=lambda x: {'analysis_start_time': 'Analysis Start Time',
+                                                                   'last_multi_scan': 'Last Multi Scan',
+                                                                   'multiscan_result': 'Multiscan Result',
+                                                                   'threat_score': 'Threat Score',
+                                                                   'verdict': 'Verdict',
+                                                                   'sha256': 'Sha256'
+
+                                                                   }.get(x, x), removeNull=True)
```

Comment:
why is the default value here is `x`?

Reply:
it's to map the headers for the table. if i have an object coming back as {'hello' : 'world', 'goodbye' : 'world'} and I want to have my headers be : hello, good-bye, I can look in the given dictionary for header mappings, but I dont need to provide the header in case I dont want it changed. So dict[goodbye] -> goodbye , dict[hello] -> Null coalesce to hello